Formula/code-server.rb

```diff
@@ -15,6 +15,7 @@ class CodeServer < Formula
   end
 
   depends_on "python@3.10" => :build
+  depends_on "bash" => :build
```

Comment:
```suggestion
  depends_on "bash" => :build
  depends_on "python@3.10" => :build
```
third try is a charm